src/main/resources/resources/logisim/strings/gui/gui.properties

```diff
@@ -581,6 +581,7 @@ simWidthErrorBackgroundTitle = Width error background color:
 simWidthErrorCaptionTitle = Width error caption color:
 simWidthErrorHighlightTitle = Width error highlight color:
 simWidthErrorTitle = Width error color:
+simClockFrequencyTitle = Clock frequency display color:
```

Comment:
Please add the new key in all property files

Reply:
Done